backend/test/modules/player-profile/core/application/PlayerProfileWriteSide.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { testPlayerProfileModule } from './TestPlayerProfileModule';
+import { CreatePlayerProfile } from '../../../../../src/modules/player-profiles/core/application/command/CreatePlayerProfile';
+import { PlayerProfileWasCreated } from '../../../../../src/modules/player-profiles/core/domain/event/PlayerProfileWasCreated';
+import { CommandResult } from '../../../../../src/shared/core/application/command/CommandResult';
+import Failure = CommandResult.Failure;
+
+describe('Player profile | Write Side', () => {
+  it('given new (no records in data base) player profile data, when create player profile, then new player profile was created', async () => {
```

Comment:
`no records in data base`
Testy mogą działać i bez bazy danych :) To czy używamy bazy czy trzymamy w pamięci to jest szczegół implementacyjny. 
Opis powinien się skupiać na działaniu, a nie na kwestaich technicznych. 

Czyli coś w stylu 
'Given player profile not created` i dalej OK :)

Reply:
'Given data of not yet created player profile, when create player profile, then new player profile was created' jeszcze trochę podrasowałem